examples/applications/slack/pipeline.yaml

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright DataStax, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+topics:
+  - name: "input-topic"
+    creation-mode: create-if-not-exists
+pipeline:
+  - name: "convert-to-json"
+    type: "document-to-json"
+    input: "input-topic"
+    configuration:
+      text-field: "text"
+  - name: "submit to slack API"
+    type: "http-request"
+    configuration:
+        url: "https://hooks.slack.com/services/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"
+        body: '{"text":"hello from langstream"}'
```

Comment:
here you can refer to the input message, this is a mustache template
https://docs.langstream.ai/building-applications/api-reference/agents#http-request